compose/bigchaindb/Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM python:3.5
+FROM python:3.6.2rc1
 
-WORKDIR /usr/src/app
+RUN apt-get update && apt-get install -y vim
 
-RUN pip install --upgrade pip
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
 
+RUN pip install --upgrade pip ipdb ipython
 RUN pip install bigchaindb
```

Comment:
Intentional removal of Docker cache here. Can cause hard to debug errors for docker users + we're not using docker on Travis for this repo, so it's OK.